permission_handler_platform_interface/lib/src/permissions.dart

```diff
@@ -137,6 +137,12 @@ class Permission {
   ///iOS: Nothing
   static const requestInstallPackages = Permission._(24);
 
+  ///Android: Nothing
+  ///iOS: Allows user to accept that your app collects data about end users
+  ///and shares it with other companies
+  ///for purposes of tracking across apps and web sites.
```

Comment:
nit: we can make this documentation a bit more compact which will make it easier to read. The rule would be to wrap the lines when they reach 80 characters:

```suggestion
  ///iOS: Allows user to accept that your app collects data about end users and
  //shares it with other companies for purposes of tracking across apps and
  ///websites.
```